generators/newfpmpage/index.js

```diff
@@ -2,6 +2,7 @@ const path = require("path");
 const Generator = require("yeoman-generator");
 const fpmWriter = require("@sap-ux/fe-fpm-writer");
 const serviceWriter = require("@sap-ux/odata-service-writer");
+const isTypescriptEnabled = require("@sap-ux/ui5-application-writer").isTypescriptEnabled;
```

Comment:
FYI - a nicer way of writing this is:

```js
const { isTypescriptEnabled } = require("@sap-ux/ui5-application-writer");
```

Reply:
Thanks, done with https://github.com/ui5-community/generator-ui5-project/pull/58/commits/696df39fb35c8b3fb42aa9e7f0dda67d5f73a58e